src/js/main.js

```diff
@@ -77,20 +77,26 @@ export class Session {
     this.store.resetAudio();
 
     const { audio } = this.store.getAll();
+    const currentTrack = this.store.getBackgroundMusic();
+
     this.effectSound.val(audio.volume.effect);
     this.bgSound.val(audio.volume.background);
 
-    $(`#${this.store.getBackgroundMusic()}`).prop('checked', true);
+    $(`#${currentTrack}`).prop('checked', true);
     this.backgroundMusic.setVolume(audio.volume.background);
     this.soundEffect.setVolume(audio.volume.effect);
 
-    this.backgroundMusic.play(this.store.getBackgroundMusic());
+    this.backgroundMusic.play(currentTrack);
+    this.backgroundMusic.playOnLoop(currentTrack);
   }
 
   subscribeEventListeners() {
     $(document).one('click', () => {
       if (this.store.getAudioIsEnabled()) {
-        this.backgroundMusic.play(this.store.getBackgroundMusic());
+        const currentTrack = this.store.getBackgroundMusic();
+
+        this.backgroundMusic.play(currentTrack);
+        this.backgroundMusic.playOnLoop(currentTrack);
```

Comment:
If you apply the suggested changes above this code can be as simple as the one below
```suggestion
        this.backgroundMusic.play(currentTrack, {loop: true});
```

Reply:
Do you have multiple background music playing at the same time? If yes then it might be a problem otherwise it should not affect. Because sound effects and background music instances are separated objects. 
However if you face the issue again feel free to leave your approach as it works